src/open_inwoner/kvk/exceptions.py

```diff
@@ -0,0 +1,21 @@
+from typing import Self
+
+from requests import Response
+
+
+class KVKAPIException(Exception):
+    def __init__(self, message=None):
+        super().__init__(message)
+        self.message = message
+
+    @classmethod
+    def from_error_response(cls, response: Response) -> Self:
+        try:
+            data = response.json()
+        except Exception:
+            api_error_message = ""
+        else:
+            api_error_message = data.get("fout", [{}])[0].get("omschrijving", "")
+        cls.message = api_error_message
+
+        return cls(message=api_error_message)
```

Comment:
I think we can simplify this, and keep the response around for traceability:

```suggestion
class KVKAPIException(Exception):
    response: Response | None
    def __init__(self, message=None, *, response: Response | None = None):
        super().__init__(message or "An error occurred while communicating with the KvK API")
        self.response = response

    @classmethod
    def from_error_response(cls, response: Response) -> Self:
        api_error_message: str | None = None
        with contextlib.suppress(BaseException):
            data = response.json()
            api_error_message = data.get("fout", [{}])[0].get("omschrijving", "")
  
        return cls(message=api_error_message, response=response)
```